example/extension/sqlinject/README.md

```diff
@@ -0,0 +1,34 @@
+# Use extensions to detect sql injection
+
+The document demonstrates how to use extensions to detect sql injection.
+
+## Step1: Replace path in config.json
+Replace the `Path` in `config.json` with the actual absolute path of `rules` directory, for example:
+``` json
+[{
+"ImportPath": "database/sql",
+"Function": "Query",
+"ReceiverType": "*DB",
+"OnEnter": "sqlQueryOnEnter",
+"Path": "/Users/liuziming/Desktop/opentelemetry-go-auto-instrumentation/example/extension/sqlinject/rules"
```

Comment:
Maybe a placeholder is enough.

Reply:
> Maybe a placeholder is enough.

It is just an example. It is a placeholder in project file.